content/en/docs/Features/ambient.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Istio Ambient Mesh
+description: Visualizing Ambient Mesh with Kiali
+---
+
+Kiali provides visualization for Ambient Mesh components: 
+
+* [Control Plane Ambient Mesh](#control-plane-ambient-mesh)
+* [Ambient nampespace](#ambient-namespace)
+* [Workloads in Mesh](#workloads-in-ambient-mesh)
+* [Ambient Telemetry](#ambient-telemetry)
+
+{{% alert color="warning" %}}
+The Kiali Ambient features, as well as Ambient Mesh, are evolving. Some of these features are in alpha status. For enhancements or detected issues, don’t hesitate to open a [GitHub issue](https://github.com/kiali/kiali/issues/new/choose). 
+{{% /alert %}}
+
+### Control Plane Ambient Mesh
+
+When the control plane is in Ambient mode, Kiali will show an Ambient label on the control plane card. 
```

Comment:
```suggestion
When the control plane is in Ambient mode, Kiali will show an Ambient badge on the Overview page control plane namespace card.  It will also be reflected in the control plane side-panel on the Mesh page.
```